config/settings.yaml.example

```diff
@@ -26,9 +26,9 @@
 # errors in your browser
 :webpack_dev_server_https: false
 
-# Ruby on Rails version (available: 4.2, 5.0)
-# Defaults to 4.2 on Ruby 2.0-2.2, or 5.0 on Ruby 2.3+
-#:rails: 4.2
+# Ruby on Rails version
+# Defaults to 5.0
+#:rails: 5.0
```

Comment:
Do we actually still want this setting and all it's related code if we only support one version?

Reply:
I find it very useful e.g. on the road from 5.0 to 5.1.